storm-server/src/main/java/org/apache/storm/scheduler/resource/strategies/scheduling/BaseResourceAwareStrategy.java

```diff
@@ -126,7 +135,9 @@ protected boolean scheduleExecutor(
         } else {
             String comp = td.getExecutorToComponent().get(exec);
             NormalizedResourceRequest requestedResources = td.getTotalResources(exec);
-            LOG.error("Not Enough Resources to schedule Task {} - {} {}", exec, comp, requestedResources);
+            schedulingFailText = "Not Enough Resources to schedule Task "
+                    + exec + " - " + comp + " " + requestedResources;
+            LOG.error(schedulingFailText);
```

Comment:
Can we make this a warn message?  It's always bothered me that this reports as an ERROR.  There's no error in the code flow, the cluster just does not have enough resources.

Reply:
Changed to warn